### PR TITLE
Nokia srl prompt stripping

### PR DESCRIPTION
**Pull request by ktbyers**

### CODE REVIEW

**Comment by ktbyers** [comment]
@jeffkala Added some tests and converted the one method to a private method (as more consistent with standard Netmiko behavior).

File: netmiko/nokia/nokia_srl.py [hunk 1]
```diff
@@ -53,6 +53,32 @@ def session_preparation(self) -> None:
...
 
+    def strip_prompt(self, *args: Any, **kwargs: Any) -> str:
+        """Strip the line1 of multiline prompt from the output."""
+        a_string = super().strip_prompt(*args, **kwargs)
+        return self._strip_context_items(a_string)
+
+    def _strip_context_items(self, a_string: str) -> str:
+        """Strip NokiaSRL-specific output.
+
+        Nokia will put extra context is line 1 of the prompt, such as:
```
Comment (hellt) [inline netmiko/nokia/nokia_srl.py:64]:
```suggestion
        Nokia will put extra context in the 1st line of the prompt, such as:
```

File: netmiko/nokia/nokia_srl.py [hunk 2]
```diff
@@ -53,6 +53,32 @@ def session_preparation(self) -> None:
             self.disable_paging(command=command, cmd_verify=True, pattern=r"#")
         self.set_base_prompt()
 
+    def strip_prompt(self, *args: Any, **kwargs: Any) -> str:
+        """Strip the line1 of multiline prompt from the output."""
```
Comment (hellt) [inline netmiko/nokia/nokia_srl.py:57]: is this statement correct about the 1st line? Shouldn't the strip tackle the two lines?

Comment (jeffkala) [reply netmiko/nokia/nokia_srl.py]: Still calling super which I'd assume would strip the original prompt.

Comment (ktbyers) [reply netmiko/nokia/nokia_srl.py]: I updated the docstrings, the `super()` call will get rid of `A:srl1#`.

The `_strip_context_items` call will get rid of things like `--{ running }...`

**Comment by ktbyers** [comment]
Look good @jeffkala @hellt ?

**Review by jeffkala — APPROVED** [review]

File: netmiko/nokia/nokia_srl.py (hunk 2 above)
Comment (hellt) [reply netmiko/nokia/nokia_srl.py]: ah, I see, by "prompt" you mean only one line of the SR Linux prompt.
Ok, lgtm

**Pull request merged**